src/utils/hints/stateChannelsHints.js

```diff
@@ -1,4 +1,5 @@
 export const stateChannelsHints = {
+  stateChannel: 'An off-chain mechanism that allow participants to conduct multiple transactions or interactions without involving the main blockchain for each step. By reducing transaction fees and increasing transaction speed, state channels provide a scalable solution for applications requiring frequent and rapid interactions while still leveraging the security of the Aeternity blockchain.',
```

Comment:
```suggestion
  stateChannel: 'State Channels allow the gas-free execution of smart contracts and transactions, privately and with the speed of light, while still being able to escalate on-chain in case of disagreement.',
```

I would use the same we have on the dashboard and I would also add a "Learn more" link to https://aeternity.com/state-channels

Reply:
Added here in other places whenever there was a dedicated documentation page. The rest of the entities are described only on medium and github RFCs (at least as per my search)

Reply:
I provided another suggestion with "A state channel" instead of "State Channels", see below